app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarActivity.java

```diff
@@ -74,14 +74,26 @@
     private CalendarController calendarController;
     private final LifecycleProvider<Lifecycle.Event> provider = AndroidLifecycle.createLifecycleProvider(this);
 
+    /**
+     * Variables for filtering calendar events
+     */
+    private boolean showCanceledEvents = true;
```

Comment:
Why does this need to be a member variable of the class? 

Better just fetch the value from shared preferences once its needed and have less state inside the object wide access (=> usually means less bugs)

Up to you - just a suggestion.

Reply:
Agree and changed it, I only thought that Utils would always fetch it from storage and that it would be nicer to have it in RAM.